client/verta/verta/registry/entities/_modelversion.py

```diff
@@ -627,14 +623,8 @@ def get_artifact(self, key):
 
         return artifact_stream
 
-    def download_artifact(self, key, download_to_path):
```

Comment:
The shared preamble code has moved to `DeployableEntity.download_artifact` with the diverging implementations following in `_internal_download_artifact`